CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- Energy Yield Analysis (EYA) to Operational Assessment(OA) Gap Analysis method
+- Uncertainty quantification
+- Implemented open source Engie example data, and complete update of example notebooks
+- Switch to standard BSD-3 Clause license
+- Automated quality control method to assist with data ingestion. Tools in this method include daylight savings time change detection and identification of the diurnal cycle.
+- Electrical losses method
```

Comment:
Another item to add is: Method for estimating long-term turbine gross energy (excluding downtime and underperformance losses)